admin/includes/modules/update_product_meta_tags.php

```diff
@@ -17,7 +17,7 @@
   if (isset($_GET['pID'])) {
     $products_id = zen_db_prepare_input($_GET['pID']);
   }
-  $products_date_available = zen_db_prepare_input($_POST['products_date_available']);
+  $products_date_available = !empty($_POST['products_date_available']) ? zen_db_prepare_input($_POST['products_date_available']) : '';
```

Comment:
I see this PR started with just fixing an undefined array index. 
But, is this `$products_date_available` variable actually even used anywhere in this workflow? 
Can't this line (and the one below it) be safely removed?

Reply:
I don't see any use of it, I assume it is some debris originally copied from update_product?
I'll remove it.